### PR TITLE
misc upgrades to README

### PR DESCRIPTION
**Pull request by SethTisue**
- link to new scala/contributors room on Gitter
- remove link to old disused CI
- replace list of mailing lists with single link to community page
- link to 2.12 nightlies too, not just 2.11
- update required JDK version info

review by: all and sundry

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +61,7 @@ scala/
 
 ## Requirements
 
-You'll need a Java SDK (6 or newer), Apache Ant (version 1.9.0 or above), and curl (for `./pull-binary-libs.sh`).
+You'll need a Java SDK (6 for 2.11.x, 8 for 2.12.x), Apache Ant (version 1.9.0 or above), and curl (for `./pull-binary-libs.sh`).
```
Comment (janekdb) [inline README.md]: There's another reference to JDK 1.6 in the IntelliJ section.

Comment (SethTisue) [inline README.md]: Ah, good catch. I've added an additional commit that fixes that, and also fixes the redundancy between the main readme and the Eclipse & IntelliJ readmes.

File: src/eclipse/README.md [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
-and navigate to `scala/src/eclipse`. Check all projects and click ok.
+The following points describe how to build Scala using Eclipse.
 
-0. You need to define a `path variable` inside Eclipse. Define `SCALA_BASEDIR` in 
-`Preferences/General/Workspace/Linked Resources`. The value should be the absolute 
+0. Download the [Scala IDE bundle](http://scala-ide.org/download/sdk.html). It comes preconfigured for optimal performance.
+
+0. Run `ant init` to download some necessary jars.
+
+0. Import the project (in `src/eclipse`) via `File` → `Import Existing Projects` and navigate to `scala/src/eclipse`. Check all projects and click ok.
```
Comment (performantdata) [inline src/eclipse/README.md:12]: Might mention that it may be desirable to do this in a new workspace, since it creates many projects.

**Comment by performantdata** [comment]
Looks good to me, except I can't judge the IntelliJ stuff.

File: src/eclipse/README.md (hunk 2 above)
Comment (SethTisue) [inline src/eclipse/README.md:12]: I lack the expertise to improve on this myself, but I invite anyone reading this to submit a followup PR with further Eclipse-specific improvements.

Comment (performantdata) [inline src/eclipse/README.md:12]: It's not terribly important.

**Comment by janekdb** [comment]
LGTM

Have not validated the Eclipse instructions.

**Comment by performantdata** [comment]
LGTM for Eclipse

**Pull request merged**